src/main/java/org/radarcns/management/domain/support/AutowireHelper.java

```diff
@@ -0,0 +1,45 @@
+package org.radarcns.management.domain.support;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by dverbeec on 17/11/2017.
+ */
+@Component
+public class AutowireHelper implements ApplicationContextAware {
```

Comment:
Doesn't the ` @Qualifier("nameOfTheInstance")` support this?

Reply:
I get the reason for custom AutowireHelper. I think it would be good if we also add the limitation of not being able to Autowire beans in JPA Event listeners to the Javadoc.

Reply:
Good point! Added a brief Javadoc to the class.